JavaScript/signin.js

```diff
@@ -0,0 +1,67 @@
+
+const registeredEmails = {
+   email: ['test@codeit.com'], 
+   password: ['codeit101']
+}
+const mailFormat = /^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/
+
+const inputtedLoginEmailCheck = document.querySelector('#loginEmailInput');
+const inputtedLoginPasswordCheck = document.querySelector('#loginPasswordInput');
+const orderCheckEmail = document.querySelector('#loginConfirmButton');
+
+
+function submitLoginData(event)  {
+   event.preventDefault();
+   let userVerification = 0
+   for (let i = 0 ; i < registeredEmails.email.length ; i++) {
+      if (inputtedLoginEmailCheck.value == registeredEmails.email[i]) {
+         if (inputtedLoginPasswordCheck.value == registeredEmails.password[i]) {
+            userVerification = true
+            window.location.replace("folder.html");
+         }
+      } 
+   }
```

Comment:
(제취향) 개인적으로는 중첩되는걸 선호하지 않습니다. 가독성 측면에서 좋지 않은 것 같아서요.
아래처럼 작업도 가능합니다.

```suggestion
   for (let i = 0 ; i < registeredEmails.email.length ; i++) {
      if (inputtedLoginEmailCheck.value !== registeredEmails.email[i]  || inputtedLoginPasswordCheck.value !== registeredEmails.password[i]) return;
        userVerification = true
        window.location.replace("folder.html");
   }
```

Reply:
```suggestion
const { email, password } = registeredEmails;
const isValidAccount = email.includes(inputtedLoginEmailCheck.value) && password.includes(inputtedLoginPasswordCheck.value)

if(!isValidAccount) return;

userVerification = true
window.location.replace("folder.html");

```

Reply:
조언대로 시인성을 높이기 위해 Boolean 형태를 반환 비교식은 최대한 변수에 할당하였습니다! 감사합니다